### DISCUSSION TITLE
Downloading HRRR from Pando Archive

### DISCUSSION THREAD

**Discussion by blaylockbk** [blaylockbk/Herbie#15] [General]
Details about the HRRR data archived on the Pando system by the University of Utah is [here](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/hrrr_FAQ.html). 

# Download from Pando (GUI)
You may download data from the [click of a button](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/cgi-bin/hrrr_download.cgi).
[![image](https://user-images.githubusercontent.com/6249613/124647124-65b15a00-de4a-11eb-898e-1051768f73c4.png)](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/cgi-bin/hrrr_download.cgi)

# Download from Pando (Command Line or Script)
For more specific details about downloading the data from the command line or script, check out the hidden page "[Scripting Tips](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/hrrr_script_tips.html)" 
[![image](https://user-images.githubusercontent.com/6249613/124646924-1e2ace00-de4a-11eb-8387-95b1a25128bf.png)](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/hrrr_script_tips.html)

---
## Download from AWS (GUI)
To download HRRR data from AWS with "[the click of a button](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/cgi-bin/generic_AWS_download.cgi?DATASET=noaa-hrrr-bdp-pds)," use my generic AWS bucket explorer
[![image](https://user-images.githubusercontent.com/6249613/124647468-db1d2a80-de4a-11eb-8459-bb94a55e5b79.png)
](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/cgi-bin/generic_AWS_download.cgi?DATASET=noaa-hrrr-bdp-pds)

Alternatively, AWS now provides a GUI with [browse bucket](https://noaa-hrrr-bdp-pds.s3.amazonaws.com/index.html).

**Comment by lukevalin** [comment]
Hi Brian,

The site you have made is excellent. I notice that HRRR prior to 2019 is no longer archived on Pando, but did not see any notes on why or if there is an alternative source - I may not have looked hard enough. Do I need to transition my web scripts to use AWS and your HRRRB python scripts?

--Luke

**Reply by blaylockbk** [reply]
Yes, you should use AWS to get your HRRR data instead of Pando.

This notice is on the [Pando HRRR page](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/hrrr_FAQ.html)
![image](https://user-images.githubusercontent.com/6249613/149195245-a7263bef-3c33-45f2-a029-0a733b4974a5.png)

Also, [hrrr.chpc.utah.edu](http://hrrr.chpc.utah.edu/) redirects to a new page with some additional information. Going forward, archived HRRR files from as early as 2014 are available through NOAA's Big Data Program on AWS, Google Cloud Platform, and Azure.

> FYI, The Herbie package is still functional to download HRRR files from AWS and other sources (see [Herbie Docs](https://blaylockbk.github.io/Herbie/_build/html/)) but comes with no guarantee. I'm no longer at the University of Utah working on this project.
